lib/Service/src/SettingsHandler.cpp

```diff
@@ -280,3 +282,16 @@ SettingsHandler::~SettingsHandler()
 /******************************************************************************
  * Local Functions
  *****************************************************************************/
+
+/**
+ * Convert a string to a json variant.
+ *
+ * @param[in]   str     String to convert.
+ * @param[out]  variant Json variant.
+ *
+ * @return If successful, it will return true otherwise false.
+ */
+bool convertToJson(const String& str, JsonVariant variant)

```

Comment:
Should be in ArduinoNative to be usable for all components.

Reply:
Where in ArduinoNative would it be best? Inside WString.h? `#include <ArduinoJson.h>` must also be included